pkg/sql/schemachanger/testdata/explain_verbose/drop_column_basic.rollback_1_of_7

```diff
@@ -72,13 +80,36 @@ EXPLAIN (ddl, verbose) rollback at post-commit stage 1 of 7;
         │       └── • skip PUBLIC → ABSENT operations
         │             rule: "skip index-column removal ops on index removal"
         │
-        └── • 8 Mutation operations
+        └── • 10 Mutation operations
             │
             ├── • SetColumnName
             │     ColumnID: 2
             │     Name: j
             │     TableID: 104
             │
+            ├── • UpsertColumnComment
+            │     ColumnID: 2
+            │     Comment: j has a comment
+            │     PGAttributeNum: 2
+            │     TableID: 104
+            │
+            ├── • LogEvent
+            │     Element:
+            │       ColumnComment:
+            │         columnId: 2
+            │         comment: j has a comment
+            │         pgAttributeNum: 2
+            │         tableId: 104
+            │     EventBase:
+            │       Authorization:
+            │         UserName: root
+            │       Statement: ALTER TABLE ‹defaultdb›.public.‹t› DROP COLUMN ‹j›
+            │       StatementTag: ALTER TABLE
+            │       TargetMetadata:
+            │         SourceElementID: 1
+            │         SubWorkID: 1
+            │     TargetStatus: 2
```

Comment:
I wonder about this. Perhaps we want to not remove the comment until we hit the non-revertible stage.

Reply:
I don't know, it seems like a side-effect we'd like to see in-txn.

Reply:
I'm working towards that, but this test output changed only because I added a comment, not because I changed any behaviour, so eager comment removal is already a thing right now.